index.html

```diff
@@ -562,21 +562,38 @@
 
-The resulting BRDF of the glossy-diffuse slab is the combination of a "glossy" specular lobe provided by reflection from the dielectric interface, and a diffuse lobe provided by bounces off the substrate. This models for example the reflection from shiny, totally opaque surfaces such as dense plastic, rock, and concrete. This form of BRDF model is classic in computer graphics and typically termed the Ashikhmin-Shirley or Kelemen model ([#Ashikhmin2000], [#Kelemen2001], [#Kulla2017], [#Kutz2021]).
+Physically there will be darkening and saturation of the observed color due to multiple inter-reflections (scattering) within the gloss layer, as described in the coat darkening section. However in this case, we wish to automatically compensate for this effect by adjusting the albedo of $f_\mathrm{diffuse}$ in order to make the observed color match the input color $\mathbf{C} =$ **`base_weight`** * **`base_color`** (in a similar spirit to the albedo remapping described in the Subsurface section).
 
-A reasonable practical albeit non-reciprocal approximation to the resulting BRDF of the glossy-diffuse slab is
+To define the meaning of the specified color in terms of the underlying base albedo, we write the normal-direction reflectance of the glossy-diffuse slab, $\mathbf{E}_\textrm{glossy-diffuse}$ in the general form
+\begin{eqnarray}
+\mathbf{E}_\textrm{glossy-diffuse} = \mathbf{E}_\textrm{spec} + \mathbf{E}_\textrm{diffuse}
+\end{eqnarray}
+where $\mathbf{E}_\mathrm{spec}$ is the normal-direction reflectance of all energy reflected from the dielectric interface without transmission, and $\mathbf{E}_\mathrm{diffuse}$ is the normal-direction reflectance of all energy transmitted through the interface, scattered in the diffuse medium, and transmitted back out.
+We then _define_ $\mathbf{C} =$ **`base_weight`** * **`base_color`** to be such that the reflectance of the remaining energy transmitted into the slab, $\mathbf{E}_\textrm{diffuse}$, is given by
+\begin{eqnarray} \label{glossy_diffuse_albedo_constraint}
+\mathbf{E}_\textrm{diffuse} = \bigl( 1 - \mathbf{E}_\textrm{spec} \bigr) \mathbf{C} \ .
+\end{eqnarray}
+In other words, the selected color $\mathbf{C}$ parametrizes the fraction of the energy transmitted into the dielectric layer that is subsequently transmitted back out due to reflection from the base interface. Thus since $0 \le \mathbf{C} \le 1$, $\mathbf{E}_\textrm{glossy-diffuse} \le 1$ so energy is always conserved, and if $\mathbf{C}=1$ then $\mathbf{E}_\textrm{glossy-diffuse}=1$ which guarantees that a white $\mathbf{C}$ passes the furnace test. According to this definition, $\mathbf{C}$ is the observed reflection color (viewed at normal incidence under uniform illumination) in areas where the Fresnel reflection is negligible, and otherwise the observed color is a blend of $\mathbf{C}$ with the gray Fresnel reflection that conserves total reflected energy.
```

Comment:
Just to clarify, in this version of the text, the base color behaves similarly to the subsurface color in that it represents the observed color. However, as discussed on Slack, we would like to modify this such that the base color instead represents the single-scattering microfacet albedo, implying that the observed diffuse color will be darker / more saturated than the specified color (when it's not white). This will be consistent with our current description of the metal component.

Reply:
I don't think that scheme (i.e. saying we "only take single-scattering into account" in $\mathbf{E}_\textrm{diffuse}$) going to work, since e.g. if the "observed" color $\mathbf{C}$ is white, there is no way for single-scattering to achieve that without violating energy conservation (since even with a white base albedo, single-scattering loses energy).

The way we set this up we demanded that if the selected/desired color $\mathbf{C}$ is white, the base must preserve energy, so cannot just omit the multiple scattering. (Also technically, Oren-Nayar doesn't cleanly split into single and multiple scattering terms like a GGX microfacet model, so it's not really clear what "single-scattering Oren-Nayar" model means).

I think the right way to say it is to say that the formulas define what the Oren-Nayar albedo has to be in the *zero roughness* case. Then given that albedo, the actual roughness is used when evaluating the BRDF. The albedo cannot increase just by increasing the roughness, since all that does is add more energy dissipation, it can only darken/saturate.

Reply:
Makes sense. That looks like a good clarification for future proofing.